src/PKSim.Presentation/Presenters/Main/BuildingBlockExplorerPresenter.cs

```diff
@@ -109,8 +109,8 @@ protected override void AddProjectToTree(IProject project)
       private ITreeNode addProtocolToTree(Protocol protocol) => addBuildingBlockToTree(protocol, PKSimRootNodeTypes.ProtocolFolder, ApplicationIcons.Protocol);
 
       private ITreeNode addObserverSetToTree(ObserverSet observerSet) => addBuildingBlockToTree(observerSet, PKSimRootNodeTypes.ObserverSetFolder, ApplicationIcons.Observer);
-      
-      private ITreeNode addExpressionProfileToTree(ExpressionProfile expressionProfile) => addBuildingBlockToTree(expressionProfile, PKSimRootNodeTypes.ExpressionProfileFolder, ApplicationIcons.ProteinExpression);
+
+      private ITreeNode addExpressionProfileToTree(ExpressionProfile expressionProfile) => addBuildingBlockToTree(expressionProfile, PKSimRootNodeTypes.ExpressionProfileFolder, ApplicationIcons.IconByName(expressionProfile.Icon));
```

Comment:
Maybe it's enough to have them like this?
![image](https://user-images.githubusercontent.com/1041237/140412250-945cf442-de6f-4a8c-a661-cb9713bd757f.png)

Reply:
I think for the most proteins we will have only 1 phenotype within a project, thus for me it seems to be enough

Reply:
Agreed. I tried with a nested structure and it was less "usable" because one always had to open multiple folders